lessons/module-2/closures.md

```diff
@@ -148,15 +157,127 @@ Our most thorough definition of a closure is now **when an inner function has ac
 
 In the previous example, you'll notice we could still technically change those grades and snoop on them if we wanted to. This is why we say JavaScript doesn't have a true fashion for creating private variables. We can kind of imply that you shouldn't be fussing with something by hiding it in a function and not exposing that variable declaration outside of it - but we can still gain access to that value. So closures aren't really going to help if you have truly sensitive data that nobody should be able to see.
 
+<section class="answer">
+### Practice in small groups
+Consider the following situation. I want to create function that helps manage rent prices, but keeps the actual rent amount private through closure. 
+
+```js
+const rentPrice = (initialRent) => {
+  // your code goes here
+}
+
+var rent = rentPrice(8000);
+  
+// Update data using private methods 
+rent.getRent() // returns 8000
+rent.incRent(2000);
+rent.decRent(1500);
+rent.decRent(1000); 
+rent.incRent(2000); 
+rent.getRent();
+```
+***Hint:*** `rentPrice` should return a series of functions for updating our confidental rent (the initial parameter).
```

Comment:
In order for students to understand what they're supposed to do in this practice example, we might want to add more direction to the instructions or the hint.

We could add this to the hint:  "Look at the functions that are being invoked using dot notation on the rent variable.  Use your problem solving skills to figure out what each of those functions is doing then build that function within the rentPrice closure function"